python/ndpi_typestruct.py

```diff
@@ -507,6 +507,15 @@ class Http2(Structure):
         ("nat_ip", c_char * 24)
     ]
 
+class FtpImapPopSmtp(Structure):
```

Comment:
How could this work before? The whole struct was missing in the flow struct. Python bindings via ctypes seems to be horribly broken.